terraform/modules/db/run-migrations.sh

```diff
@@ -0,0 +1,8 @@
+# Run Cloud SQL Auth Proxy in background, run migrations, kill proxy.
+cloud-sql-proxy $DB_CONNECTION_NAME --credentials-file $CREDENTIALS_FILE &
```

Comment:
What if host machine doesn't have cloud-sql-proxy?

Asking because sometimes scripts/provisioners can corrupt state if they don't run correctly

Reply:
Addressed in README.